pyband/wallet.py

```diff
@@ -242,3 +245,28 @@ def to_cons_bech32(self) -> str:
     def to_hex(self) -> str:
         """Return a hex representation of address"""
         return self.addr.hex()
+
+
+class Ledger:
+    def __init__(self, **kwargs):
+        if kwargs.get("derivation_path", None) is None:
+            hid_path = DEFAULT_LEDGER_DERIVATION_PATH
+        else:
+            hid_path = kwargs.get("path")
+
+        if kwargs.get("cosmos_app", None) is None:
+            self.cosmos_app = CosmosApp(bip44_to_list(hid_path))
+        else:
+            self.cosmos_app = kwargs.get("cosmos_app")
+
+    def disconnect(self):
+        self.cosmos_app.disconnect()
+
+    def app_info(self):
+        return self.cosmos_app.get_version()
+
+    def sign(self, msg: bytes):
+        return self.cosmos_app.sign_secp256k1(msg)
+
+    def get_pub_key_and_bech32_address(self):
```

Comment:
Should separate to get_public_key and get_address function and return PublicKey and Address Type to compatible with pyband codebase.